javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/config/AgentConfig.java

```diff
@@ -11,20 +11,14 @@ public final class AgentConfig {
 
   public static boolean isInstrumentationEnabled(
       ConfigProperties config, Iterable<String> instrumentationNames, boolean defaultEnabled) {
-    // If default is enabled, we want to enable individually,
-    // if default is disabled, we want to disable individually.
```

Comment:
maybe update the comment to reflect this change? something like use default config only when system properties are not set.